challenge.rb

```diff
@@ -1,26 +1,27 @@
 
-def capitalize_each_string(input)
+def capitalize_each_string(*animal)
   #implement your solution here
+  animal.each { |a| puts a.capitalize }
```

Comment:
You have the following error on this line:

```
challenge.rb:4:in `block in capitalize_each_string': undefined method `capitalize' for ["cat", "moose", "dog", "bird"]:Array (NoMethodError)
	from challenge.rb:4:in `each'
	from challenge.rb:4:in `capitalize_each_string'
	from challenge.rb:31:in `<main>'
```

Try to figure out what's wrong, but let me know if you need help.